ansible/roles/ocp4-workload-integreatly/files/threescale/create-tenant.yml

```diff
@@ -0,0 +1,81 @@
+---
+
+# set internal util variables
+
+- set_fact:
+    _tenant_id: "{{ tenant_id_format | format(item|int) }}"
+    _tenant_count: "{{ item }}"
+    _tenant_admin_username: "{{ lab_users_name_format | format(item|int) }}"
+    _tenant_admin_password: "{{ lab_users_name_format | format(item|int) }}-password"
```

Comment:
I think this will be generated, same as we did with RHMI 1.x workshops.

Reply:
could implement [this](https://github.com/redhat-cop/agnosticd/blob/development/ansible/roles/ocp-workload-integreatly/tasks/workload.yml#L9L13) here to set the password.